application/modules/quotes/models/Mdl_quotes.php

```diff
@@ -217,13 +217,8 @@ public function copy_quote($source_id, $target_id)
 
         // Copy the custom fields
         $this->load->model('custom_fields/mdl_quote_custom');
-        $custom_fields = $this->mdl_quote_custom->where('quote_id', $source_id)->get()->result();
-
-        $form_data = array();
-        foreach ($custom_fields as $field) {
-            $form_data[$field->quote_custom_fieldid] = $field->quote_custom_fieldvalue;
-        }
-        $this->mdl_quote_custom->save_custom($target_id, $form_data);
+        $db_array = $this->mdl_quote_custom->where('quote_id', $source_id)->get()->row_array() ?? [];
+        $this->mdl_quote_custom->save_custom($target_id, $db_array);
```

Comment:
If it is an empty array [] you're still saving that empty array to the custom_fields for the quote?
Why?

Reply:
@nielsdrost7 I think we have the same question. I believe that a slice of the original code was accidentally left out (see: https://github.com/InvoicePlane/InvoicePlane/pull/932#discussion_r1527173466)

Reply:
@naui95 My apologies. 
Here I indeed made a stupid mistake and accidentally modified the code incorrectly. 
In a moment I will restore the code (as I had previously modified it)...